src/app/newmarket/page.tsx

```diff
@@ -79,27 +81,88 @@ const NewmarketForm: React.FC = () => {
     return null;
   }
 
+   const styleBox = css`
+  margin: 0 auto;
+  padding: 50px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+`;
+  
+  const styleInput = css`
+    width: 100%;
+    max-width: 600px;
+    padding: 8px;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    font-size: 16px;
+    margin-bottom: 16px;
+`;
+
+  const exchangeItemStyle = css`
+  width: 168px;
+  flex: 1 1 auto;
+  max-width: 200px;
+  padding: 16px;
+  background-color: rgb(246, 246, 246);
+  border-radius: 8px;
+  text-align: center;
+  cursor: pointer;
+  transition: transform 0.3s, box-shadow 0.3s;
+  margin-bottom: 16px;
+`;
+  
+  const activeStyle = css`
+  border-color: #007bff;
+  box-shadow: 0 0 10px rgba(0, 123, 255, 0.5);
+`;
+  
+  const choiceStyle = css`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+`;
+  
+  const titStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+  
+  const descStyle = css`
+  margin-top: 8px;
+  font-size: 0.9em;
+`;
+  const imgStyle = css`
+  width: 22px;
+  margin-right: 5px;
+`;
+
+
   return (
```

Comment:
You can use a variable to contain all CSS, just like before.

Reply:
done